core/domain/email_manager.py

```diff
@@ -163,6 +163,28 @@ def log_new_error(*args, **kwargs):
 
 
 def _require_sender_id_is_valid(intent, sender_id):
+    """Only allow emails as specified in the config.
+
+    Many emails are only allowed to be sent by certain user
+    or type of user, e.g. 'admin' or an admin/moderator. This
+    function will raise an exception if the email is not allowed.
```

Comment:
if the email is not allowed --> if the given sender is not allowed to send this type of email.

(more explicit)